exercises/protein-translation/protein_translation_test.go

```diff
@@ -102,71 +89,115 @@ func TestCodon(t *testing.T) {
 		}
 		t.Logf("PASS: Protein translation test: %s", test.input)
 	}
+
+	for _, test := range codonInvalidBaseCases {
+		actual, err := FromCodon(test.input)
+		if !errors.As(err, &ErrInvalidBase{}) {
+			t.Fatalf("FAIL: Protein translation test: %s\nExpected type error: %T\nActual type error: %T",
+				test.input, ErrInvalidBase{}, err)
+		}
+		if actual != test.expected {
+			t.Fatalf("FAIL: Protein translation test: %s\nExpected: %s\nActual: %s",
+				test.input, test.expected, actual)
+		}
+		t.Logf("PASS: Protein translation test: %s", test.input)
+	}
+
+	for _, test := range codonStopCases {
+		actual, err := FromCodon(test.input)
+		if !errors.As(err, &ErrStop{}) {
+			t.Fatalf("FAIL: Protein translation test: %s\nExpected type error: %T\nActual type error: %T",
+				test.input, ErrStop{}, err)
+		}
+		if actual != test.expected {
+			t.Fatalf("FAIL: Protein translation test: %s\nExpected: %s\nActual: %s",
+				test.input, test.expected, actual)
+		}
+		t.Logf("PASS: Protein translation test: %s", test.input)
```

Comment:
We don't need to log test passes; log output will only be seen on failure.

Reply:
I agree, but this was already here, I think the idea was to print the tests that had passed when another one failed. But I agree and I will remove it.